### PR TITLE
add README documentation

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
@gwaybio This is ready for review!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
-# NF1_SchwannCell_data
\ No newline at end of file
+# NF1 Schwann Cell Data Project
+
+## Data
+
+The data used in this project is a modified Cell Painting assay on [schwann cells](https://www.ncbi.nlm.nih.gov/books/NBK544316/) from patients with [Neurofibromatosis type 1 (NF1)](https://medlineplus.gov/genetics/condition/neurofibromatosis-type-1/).
```
Comment (gwaybio) [inline README.md]:
```suggestion
The data used in this project is a modified Cell Painting assay on [Schwann cells](https://www.ncbi.nlm.nih.gov/books/NBK544316/) from patients with [Neurofibromatosis type 1 (NF1)](https://medlineplus.gov/genetics/condition/neurofibromatosis-type-1/). 
```

Schwann is a guy! https://en.wikipedia.org/wiki/Theodor_Schwann

File: README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+
+## Data
+
+The data used in this project is a modified Cell Painting assay on [schwann cells](https://www.ncbi.nlm.nih.gov/books/NBK544316/) from patients with [Neurofibromatosis type 1 (NF1)](https://medlineplus.gov/genetics/condition/neurofibromatosis-type-1/). 
+In this modified Cell Painting, there are three channels:
+
+- `DAPI` (Nuclei)
+- `GFP` (Endoplasmic Reticulum)
+- `RFP` (Actin)
+
```
Comment (gwaybio) [inline README.md:11]: Can you add an example image?? that would be so nice :)

File: README.md [hunk 3]
```diff
@@ -1 +1,32 @@
...
+There are two genotypes of the NF1 gene in these cells:
+
+- Wild type (`WT`): In column 6 from the plate (e.g C6, D6, etc.)
+- Heterozygous (`Het`): In column 7 from the plate (e.g C7, D7, etc.)
+
+## Goal
+
+The goal of this project is to predict NF1 genotype from schwann cell morphology.
```
Comment (gwaybio) [inline README.md]:
```suggestion
The goal of this project is to predict NF1 genotype from Schwann cell morphology. 
```

File: README.md [hunk 4]
```diff
@@ -1 +1,32 @@
...
+
+There are two genotypes of the NF1 gene in these cells:
+
+- Wild type (`WT`): In column 6 from the plate (e.g C6, D6, etc.)
+- Heterozygous (`Het`): In column 7 from the plate (e.g C7, D7, etc.)
+
+## Goal
+
+The goal of this project is to predict NF1 genotype from schwann cell morphology. 
+This can be done by using cell image analysis and representation learning to discover a biomarker from a pilot dataset that can be used for making predictions in future datasets.
```
Comment (gwaybio) [inline README.md]:
```suggestion
We apply cell image analysis to Cell Painting images and use representation learning to extract morphology features.
We will apply machine learning to the morphology features to discover a biomarker of NF1 genotype. 
```

File: README.md (hunk 1 above)
Comment (gwaybio) [reply README.md]: It might also be good to state _why_ studying Schwann cells in NF1 is important 🤔

File: README.md [hunk 5]
```diff
@@ -1 +1,32 @@
...
+
+## Repository Structure
+
+| Module | Purpose | Description |
+| :---- | :----- | :---------- |
+| [0_download_data](0_download_data/) | Download NF1 pilot data | Download 96 images of the NF1 pilot data  for analysis|
+| [1_preprocessing_data](1_preprocessing_data/) | Perform Illumination Correction (IC) | Use `BaSiCPy` to perform IC on images per channel |
+| [2_segmenting_data](2_segmenting_data/) | Segment Objects | Perform segmentation using `Cellpose` and outputing center (x,y) coordinates for each object |
+| [3_extracting_features](3_extracting_features/) | Extract features | Use center (x,y) coordinates in `DeepProfiler` to extract features from all channels |
+| [CellProfiler_pipelines](CellProfiler_pipelines/) | Perform a full pipeline on NF1 data using CellProfiler | Using `CellProfiler`, two pipelines are ran where one perform illumination correction and the second performs segmentation and feature extraction |
```
Comment (gwaybio) [inline README.md]:
```suggestion
| [CellProfiler_pipelines](CellProfiler_pipelines/) | Perform a full pipeline on NF1 data using CellProfiler (from IC to feature extraction) | We run two `CellProfiler` pipelines (1. illumination correction; and 2) segmentation and feature extraction) |
```

**Review by gwaybio — APPROVED** [review]
Looks great! Only minor suggestions

File: README.md (hunk 2 above)
Comment (jenna-tomkinson) [reply README.md:11]: Done deal!

**Comment by jenna-tomkinson** [comment]
@gwaybio I will merge this PR now!

**Pull request merged**